Dockerfile

```
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
+
+# Update package lists and install necessary dependencies
+RUN apt-get update && apt-get install -y \
```

```suggestion
RUN apt-get update && apt-get dist-upgrade -y && apt-get install -y \
```

To install (security) fixes